src/Integration.Vsix.UnitTests/SonarLintTagger/TaggerProviderTests.cs

```diff
@@ -223,10 +282,34 @@ public void CreateTagger_close_last_tagger_should_unregister_tracker()
             // Remove one tagger -> tracker should still be registered
```

Comment:
why is provider.ActiveTrackersForTesting.Count never 2 in this test? even though we add 2 trackers to it?

Reply:
We don't - we add two taggers.

There is one _tracker_ per document. Each tracker will have one or more _taggers_.
When all of the taggers have been removed, the tracker will unregister itself from the TaggerProvider.